.github/workflows/unitctl.yml

```diff
@@ -10,6 +10,13 @@ on:
       - master
     tags:
       - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+-[0-9]+'
```

Comment:
I don't know if we need it for *both* of these.

The `-1` is a packaging thing and ideally will go away in the future...